apps/api/src/server/app/repositories/review-questionnaire.repository.js

```diff
@@ -0,0 +1,25 @@
+import DatabaseFactory from './database.js';
+
+const newReviewRepository = (function() {
+	/**
+    * @returns {object} connection to database
+    */
+	function getPool () {
+		return DatabaseFactory.getInstance().pool;
+	}
+
+	return {
+		addReview: function(appealId, complete, reason, description) {
+			return getPool().reviewQuestionnaire.create({
+				data: {
+					appealId: appealId,
+					complete: complete,
+					...reason,
+					...description
+				}
+			});
+		},
```

Comment:
Why are we separating the `reason` and the `description` parameters here?

Reply:
realised this, and also that the double curly brackets were wrong... corrected